boards/risc-v/esp32c3/esp32c3-devkit-rust-1/src/esp32c3_apa102.c

```diff
@@ -0,0 +1,109 @@
+/****************************************************************************
+ * boards/risc-v/esp32c3/esp32c3-devkit-rust-1/src/esp32c3_apa102.c
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+
+#include <stdio.h>
+#include <stdbool.h>
+#include <debug.h>
+#include <errno.h>
+
+#include <nuttx/arch.h>
+#include <nuttx/board.h>
+#include <nuttx/spi/spi.h>
+#include <nuttx/lcd/lcd.h>
+#include <nuttx/lcd/apa102.h>
+
+#include "esp32c3_gpio.h"
+#include "esp32c3_spi.h"
+#include "esp32c3-devkit-rust-1.h"
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+#define LCD_SPI_PORTNO        2   /* On SPI2 */
+
+#ifndef CONFIG_LCD_CONTRAST
+#  define CONFIG_LCD_CONTRAST 60
+#endif
+
+/****************************************************************************
+ * Private Data
+ ****************************************************************************/
+
+struct spi_dev_s *g_spidev;
+struct lcd_dev_s *g_lcddev;
```

Comment:
Do we need `static` here?

Reply:
Either that, or these shouldn't be placed under "Private Data"